src/components/ButtonWithLoader.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import { EOS_LOADING_ANIMATED } from 'eos-icons-react'
+
+const ButtonWithLoader = ({ onClick, children, ...rest }) => {
+  const [isLoading, setLoading] = useState(false)
+  const handleClick = async () => {
+    setLoading(true)
+    await onClick()
+    setLoading(false)
+  }
+  return (
+    <button onClick={handleClick} disabled={isLoading} {...rest}>
```

Comment:
Can you please elaborate the motive of doing so because I think it should be disabled in all cases whenever loading is happening and this is happening through this code.

Reply:
So personalising it everywhere used would not benefit. If some alteration is needed please give your take on this.
@kailash360